addons/website_forum/models/forum_post.py

```diff
@@ -602,11 +603,12 @@ def mark_as_offensive_batch(self, key, values):
         _logger.info('User %s marked as spams (in batch): %s' % (self.env.uid, spams))
         return spams.mark_as_offensive(reason_id)
 
-    def bump(self):
+    def bump(self, force=False):
         """ Bump a question: trigger a write_date by writing on a dummy bump_date
-        field. One cannot bump a question more than once every 10 days. """
+        field. One cannot bump a question more than once every 10 days,
+        unless a user add a comment or a reply (force=True)"""
         self.ensure_one()
-        if self.forum_id.allow_bump and not self.child_ids and (datetime.today() - datetime.strptime(self.write_date, tools.DEFAULT_SERVER_DATETIME_FORMAT)).days > 9:
+        if force or (self.forum_id.allow_bump and not self.child_ids and (datetime.today() - datetime.strptime(self.write_date, tools.DEFAULT_SERVER_DATETIME_FORMAT)).days > 9):
```

Comment:
You are allowing people to force_write as sudo on any post in RPC ? I bet you don't want that.